objiotracing/lib/simulate.go

```diff
@@ -0,0 +1,143 @@
+package lib
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/pebble/objstorage/objstorageprovider/objiotracing"
+	"github.com/dgryski/go-clockpro"
+	"github.com/dgryski/go-s4lru"
+	"github.com/dgryski/go-tinylfu"
+)
+
+type ReplacementPolicy int
+
+const (
+	ClockPro ReplacementPolicy = iota
+	S4LRU
+	TinyLFU
+	// TODO(josh): Implement.
+	LRU
```

Comment:
Can copy from https://github.com/dgryski/go-tinylfu/blob/master/lru.go

Reply:
I will update the TODO for now, so we don't lose track of this.